plugin/vim/quick-lint-js.vim/lua/lspconfig/quick_lint_js.lua

```diff
@@ -0,0 +1,30 @@
+local configs = require("lspconfig/configs")
```

Comment:
Nit: Add copyright information. Copy-paste from other files. Also, add .lua to tools/check-file-copyrights.